torch/overrides.py

```diff
@@ -1194,6 +1194,7 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         Tensor.tile: lambda self, *reps: -1,
         Tensor.to: lambda self, dtype, non_blocking=False, copy=False, memory_format=torch.preserve_format: -1,
         Tensor.to_dense: lambda self: -1,
+        Tensor._to_dense: lambda self: -1,
```

Comment:
```suggestion
        Tensor.to_dense: lambda self, dtype=None: -1,
        Tensor._to_dense: lambda self, dtype=None: -1,
```
?

Reply:
Good catch. I'm surprised the test still runs.